community-plugins.json

```diff
@@ -605,5 +605,20 @@
     ],
     "chat_prompt": "You are Nik Shevchenko, 24yo founder of Based Hardware. Nik built Friend AI necklace that transcribes your conversations and gives you summaries and personalized advice. Nik also created openglass that takes pictures every few seconds and gives you personalized AI. Nik is a straightforward Russian guy. Nik responds in a short format, getting directly to the point. As a first message, Nik likes to write diss tracks. At the beginning of each conversation, respond with a diss track about the user",
     "deleted": false
+
+     {
+   "id":"class-notes",
+   "name":"Class Notes",
+   "author":"Findirfin",
+   "description":"Takes notes optimised for classses and lectures",
+   "image":"/plugins/logos/class-notes.jpg",
+   "memories":false,
+   "chat":true,
+   "capabilities":[
+      "chat"
+   ],
+   "memory_prompt":"You will be given a transcript of a class or lecture. Your job is to accurately identify and extract these four key points of information. 1: An outline of the main points and subpoints of the lecture. 2: A list of Important facts and concepts that need to be remembered 3: A list of homework and other assigned tasks to be completed or worked on later 4: Any other important items mentioned that do not fit into the above categories. Do not output anything for them. Label these sections, 'Class Outline', 'Important Facts and Concepts', 'Homework', and 'Other Notes'. If there is no applicable content for any of these categories, do not make anything up, just insert 'no applicable content' under the heading.",
```

Comment:
Hey @findirfin Thank you for your first contribution!

Just a small change, if your plugin has chat=true, your prompt should be in the fieled "chat_prompt" and be more related to personality.

In this case I think you are trying more of a memory prompt, in that case, please set chat: false, and capabilities, memories, instead of chat, thanks!

Reply:
Sorry about that! It should be fixed now.